src/types/responses/SanitizedBackedCandidate.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright 2017-2023 Parity Technologies (UK) Ltd.
+// This file is part of Substrate API Sidecar.
+//
+// Substrate API Sidecar is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import {
+	ISanitizedBackedCandidateExplicitValidityVote,
+	ISanitizedBackedCandidateImplicitValidityVote,
+} from './SanitizedBackedCandidateValidityVotes';
+import { ISanitizedCandidate } from './SanitizedCandidate';
+
```

Comment:
Each of these `ISanitized*` types can be placed under `src/types/responses/BlockXCMMessages.ts`.

Reply:
Actually scratch that, this is a small nit dont worry about it. You can keep it as it is.